backend/src/main/java/fk/prof/backend/worker/BackendDaemon.java

```diff
@@ -73,14 +78,15 @@ private AggregationWindowPlannerStore buildAggregationWindowPlannerStore() {
     return new AggregationWindowPlannerStore(
         vertx,
         configManager.getBackendId(),
-        config().getInteger("aggregation.window.duration.mins", 30),
+        config().getInteger("aggregation.window.duration.secs", 1800),
         config().getInteger("aggregation.window.end.tolerance.secs", 120),
         config().getInteger("policy.refresh.offset.secs", 300),
         config().getInteger("scheduling.buffer.secs", 30),
         config().getInteger("work.assignment.max.delay.secs", 120),
         workSlotPool,
         activeAggregationWindows,
-        this::getWorkFromLeader);
+        this::getWorkFromLeader,
+        this::serializeAndPersistAggregationWindow);
```

Comment:
Basically, we need to decide what to do in case we are not able to talk to S3. In such case we can either tell leader about it thru load, or commit suicide or something else clever (multipathing etc would come here).

We should just file a github issue and move on for now, mark it for milestone 2 or something.